lib/src/table/table.models.ts

```diff
@@ -0,0 +1,57 @@
+/** The type of a data the values in a column represent  */
+export type TableHeaderListValueType = "number" | "string" | "date" | "datetime" | "bool";
```

Comment:
i think another type should be added, many tables in SEB will have a superior sign icon to say that the row is clickable, i think we should care for this, do you think it will be good to have **any** as a possible type for the headers column

Reply:
I'm not sure I follow you. What do you men by superior sign icon? As is it now, this value determines the type of data display within one columns. It will default to string.

Reply:
this is what i meant, the last column of the table is an image, having an arrow means that the row is clickable, it can also be any other icon
![image](https://user-images.githubusercontent.com/14928743/84221553-18a86a80-ab08-11ea-91c3-a4f0f81e5b6f.png)

Reply:
oh I see. Maybe one of the options can be any custom html? So in the TD switch statement I would inject the innerHTML if the TableHeaderListValueType is custom-html ?

Reply:
Ok I implemented the custom html so this UI would be possible using that feature